llvm/test/Transforms/InferAlignment/ptrmask.ll

```diff
@@ -1,6 +1,8 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
 ; RUN: opt < %s -passes=infer-alignment -S | FileCheck %s
 
+target datalayout = "p1:64:64:64:32-p2:64:64:64:128"
```

Comment:
I'm not sure a index size > pointer bitwidth makes much sense, but testing that this work sounds good to me.

Reply:
I did wonder whether we should make index size > pointer size an invalid data layout...

Reply:
I'd be happy with that change - I believe I had that as part of https://reviews.llvm.org/D135158 but probably best done in a separate PR.